### ISSUE TITLE
Show total amount of hours, price & internal price

### ISSUE THREAD

**Issue by xstable** [kimai/kimai#4270]
### Describe the problem

Would be awesome to have a row with `total` at the bottom of the reporting-table. 

It could show the total of duration and prices. That would give a huge benefit to see with one view important values for the chosen time range.

### Describe the solution you'd like

![image](https://github.com/kimai/kimai/assets/1778068/5d072716-4457-4ca7-895d-704a52eab6f5)

**Comment by owlysk** [comment]
+1

This would be great and will save me a lot of time.

**Comment by EduNetEurope** [comment]
Technically, the feature is there. 

- Reporting: We can switch between total amount of hours, price and internal price. It would be really nice to view all three at the same time.
- Invoice view: We can filter for dates, clients, projects, activities, users and teams. We get totals for hours and price.

It would be super super nice to see totals in the tables as well.
For now, using the Invoice feature (preview) can be a workaround.

**Comment by EduNetEurope** [comment]
"All times" would be much improved by added totals that are updated depending on the filters selected.
It would make our controlling so much easier.
@kevinpapst: Any thoughts?

**Comment by xstable** [comment]
> For now, using the Invoice feature (preview) can be a workaround.

I don't use invoices at all with kimai, as I have an external financial booking software.

**Comment by EduNetEurope** [comment]
> > For now, using the Invoice feature (preview) can be a workaround.
> > I don't use invoices at all with kimai, as I have an external financial booking software.

Same here. We only log project hours, we don't invoice anything.

**Comment by EduNetEurope** [comment]
Maybe we could add "All times" to Reporting and allow the user the select the filters to create "weekly view for one user", "monthly view" for one user, etc.
The function is there, in "Reporting" we can select filtered views.
It is probably more complex than I can imagine, but wouldn't it be primarily an UI change?

**Comment by suther** [comment]
What is the state of this issue? Today I had need it heavily as I've searched for a given "search-term" and need to see a total amount for it.
I had do calculate it manually one by one for each day. :(

**Comment by kevinpapst** [comment]
> I had do calculate it manually one by one for each day. :(

Daily stats can be enabled in your user profile. This issue is about a total row in the listing view.

**Comment by suther** [comment]
Yep, that's what I need. A total row at the bottom which summarize the single days.
For example: 
Search String "Issue: 1234 BimBom" result in 12 entries ...
I currently see a daily summarize (if there are more then one entry per day this is very helpful!)
But to have an summarize of all single days to a overall total of current page (like in screenshot above) would be awesome.

**Comment by EduNetEurope** [comment]
For us, seeing the total amount in a filtered view would be super helpful as well.

**Comment by EduNetEurope** [comment]
Unfortunately, the feature has not been added to the roadmap yet.

@suther @xstable @owlysk @andreaspollak:
Would it make sense to request an offer and discuss the possibility of sharing the costs?
Would you be open to financially support its development?
